README.md

```diff
@@ -257,6 +268,7 @@ As always you need to navigate to the root folder of this repository first. Make
     tobiwo/eth-duties:latest \
     --validators 123456 456789 \
     --validators 0x98... \
+    --validators "111;My_Validator" "222;Validator2" \
```

Comment:
either choose comma or space separation. or should this show that both ways are possible?